### ISSUE TITLE
Touch gesture support Carousel

### ISSUE THREAD

**Issue by BHuys** [kaliber5/ember-bootstrap#613]
Good afternoon, is there or will there be any touch gesture support for the Carousel component? I would like to use gestures, swipe left/right to change the current slide etc.

**Comment by simonihmig** [comment]
No, it has not, as even the original Bootstrap implementation does not support this AFAIK. I won't have the time to implement it by myself, but I am certainly happy for a PR!

**Comment by musaffa** [comment]
As of v4.2, Bootstrap supports touch gesture. twbs/bootstrap#25776

**Comment by sergchernata** [comment]
This would be pretty nice.

**Comment by jelhan** [comment]
Happy to review and merge a pull request if someone has time to work on it.
